lib/diagnostic.js

```diff
@@ -7,6 +7,13 @@
 // `let minimum = array.reduce(min);`
 const min = function (previous, current) {
 
+  let result = arguments[0]
```

Comment:
We were looking for you to use an array iteration method rather than a `for loop` here, but this definitely works.